src/services/api/AxiosInstance.ts

```diff
@@ -37,8 +37,11 @@ const requestFormatter = async (axiosConfig: AxiosRequestConfig) => {
           },
         },
       })
+      const requestMethod = axiosConfig.method
+        ? axiosConfig.method
+        : "undefined method"
```

Comment:
If `axiosConfig` is `undefined`, accessing its property `method` could result in a type error. Could we make it more robust and concise like:
`const requestMethod = axiosConfig?.method ?? "undefined method";`

Reply:
simplified ternary operator to `??`